### ISSUE TITLE
Crash setting up submodel

### ISSUE THREAD

**Issue by JMHaro** [xLightsSequencer/xLights#1343]
I just received a crash setting up a model.  I don't know if I was doing it incorrectly.  Attached is the log file with my explanation of what I was doing in it.  I'm also sending a screen shot of what it looked like just after the error occurred (added to the zip file).  If you need any further information let me know.

[xLights_dbgrpt-18464-20181123T215316.zip](https://github.com/smeighan/xLights/files/2611997/xLights_dbgrpt-18464-20181123T215316.zip)

**Comment by keithsw1111** [comment]
I believe this will be fixed in the next release.

**Comment by JMHaro** [comment]
Just for an added note, I did figure out what I did wrong to cause the crash.  The model actually had only 110 nodes.  When I was trying to identify the particular nodes to put into the sub model, I accidentally was adding the channels between the nodes to come up with the node positions.  So the last two nodes I entered were beyond the range of the actual nodes in the string.  So hopefully that will help to identify why it crashed to to help prevent it in the future.  Thanks,
